apps/builder/app/builder/builder.tsx

```diff
@@ -3,7 +3,7 @@ import { type Publish, usePublish, useSubscribe } from "~/shared/pubsub";
 import type { Build } from "@webstudio-is/project-build";
 import type { Project } from "@webstudio-is/project";
 import { theme, Box, type CSS, Flex, Grid } from "@webstudio-is/design-system";
-import type { AuthPermit } from "@webstudio-is/trpc-interface";
+import type { AuthPermit } from "@webstudio-is/trpc-interface/server";
```

Comment:
I think type should be available in non server route. Otherwise auto import can give us many issues.

Reply:
Right now there is no `@webstudio-is/trpc-interface` according to `exports` (it was like that before my change). It was working accidentally because the root fields were desynchronized from `exports`. So with the current state of this PR the autocomplete should work fine since the lang server doesn't recognize `@webstudio-is/trpc-interface` as a valid import source